spec/support/pages/admin/split_index_page.rb

```diff
@@ -1,6 +1,6 @@
 class AdminSplitIndexPage < SitePrism::Page
   set_url '/admin'
 
-  element :splits_table, 'body.splits.index table'
-  element :log_out, '.performer-nav a[href="/admins/sign_out"]'
+  element :splits_table, 'body.AdminSplits--index table'
+  element :log_out, 'header a[href="/admins/sign_out"]'
```

Comment:
Can we just remove `.performer-nav|header` from the selector? No need to be this specific, right?